### PR TITLE
Exclude 3Di scenarios from raster layer and raster source tables

### PR DESCRIPTION
**Pull request by hoanphungt**

### CODE REVIEW

File: src/components/TableStateContainer.tsx [hunk 1]
```diff
@@ -23,9 +23,10 @@ interface Props {
   textSearchBox?: boolean; // default true
   newItemOnClick: () => void | null;
   queryCheckBox: {text: string, adaptUrlFunction: (url:string)=>string} | null;
+  scenarioIsNull?: boolean;
```
Comment (Derryrover) [inline src/components/TableStateContainer.tsx:26]: @hoanphungt 
Hi Hoan,
I am not sure about this change.
Although this works,  it does not scale up well.
What if in another table we need another default url parameter? We need to keep adding props.
I think url parameters that are standard for a certain table should either be appended as string to "baseUrl" or be passed as a newly to be made props called something like "defaultUrlParameters". 
Please let me know what you think
kr Tom

Comment (hoanphungt) [reply src/components/TableStateContainer.tsx:26]: Ah you are right. I will improve it this way.

**Comment by hoanphungt** [comment]
@Derryrover 
Could you review it again?

**Review by Derryrover — APPROVED** [review]
Excellent !

**Pull request merged**